lib/python/qmk/cli/painter/make_font.py

```diff
@@ -61,21 +58,73 @@ def painter_convert_font_image(cli):
         return
 
     # Work out the text substitutions for rendering the output data
+    subcommand = sys.argv[1]  # hacky way of telling apart painter-convert-font and painter-convert-font-image
```

Comment:
This is how [milc does it](https://github.com/clueboard/milc/blob/88b68356aafe292bdf10a045797ace2ab2aa89af/milc/milc.py#L679)... kinda meh.
```suggestion
    subcommand = cli.args.entrypoint.__name__.replace('_', '-')
```

Reply:
Yeah, still not great but definitely better than my hacky method.

I recall trying (without success) to find some API, or manully tracing down but in a better way... but i obviously missed this one, thx.